contracts/protocol/clients/voucher/BosonVoucher.sol

```diff
@@ -156,6 +157,7 @@ contract BosonVoucher is IBosonVoucher, BeaconClientBase, OwnableUpgradeable, ER
     ) external onlyRole(PROTOCOL) {
         // Make sure range start is valid
         require(_start > 0, INVALID_RANGE_START);
+        require(_start <= type(uint256).max - _length, INVALID_RANGE_LENGTH);
```

Comment:
Is the reason we are using <= and not merely < because `type(uint256).max` is a reserved magic number ?

Reply:
`<=` allows that max unit is actually reached (even though practically will never be).
If we used `<`, the last range would need to end at `type(uint256).max-1`

So  `<=` actually makes `type(uint256).max` just completely ordinary number (while `<` would make it special).